ocaml/idl/datamodel_errors.ml

```diff
@@ -452,7 +452,7 @@ let _ =
   error Api_errors.host_master_cannot_talk_back [ "ip" ]
     ~doc:"The master reports that it cannot talk back to the slave on the supplied management IP address." ();
   error Api_errors.host_unknown_to_master [ "host" ]
-    ~doc:"The master says the host is not known to it. Perhaps the Host was deleted from the master's database, the slave is pointing to the wrong master, or a pool secret rotation failed?" ();
+    ~doc:"The master says the server is not known to it. Perhaps the server was deleted from the master's database, or is pointing to the wrong master, or a pool secret rotation failed?" ();
```

Comment:
I think the following would read a bit better (i.e. wording it in terms of some sanity checks the user can do)

```
The master says the server is not known to it. Is the server in the master's database and pointing to the correct master? Did a pool secret rotation fail?
```

Apologies for not doing this properly when I touched it last

Reply:
As discussed, I corrected the message to state the sanity check in terms of the result rather than the action causing it.